src/components/block-css/index.js

```diff
@@ -219,15 +221,32 @@ const BlockCss = props => {
 		valueTabletHover,
 		valueTabletParentHover
 
-	const valueDesktop = getValue( attrName, 'desktop', 'normal' )
-	if ( hasHover ) {
-		valueDesktopHover = getValue( attrName, 'desktop', 'hover' )
-	}
-	if ( hasParentHover ) {
-		valueDesktopParentHover = getValue( attrName, 'desktop', 'parent-hover' )
+	// Some BlockCss components do not have the responsive prop. This is default behavior of obtaining
+	// the desktop value.
+	if ( responsive === false ) {
+		valueDesktop = getValue( attrName, 'desktop', 'normal' )
+		if ( hasHover ) {
+			valueDesktopHover = getValue( attrName, 'desktop', 'hover' )
+		}
+		if ( hasParentHover ) {
+			valueDesktopParentHover = getValue( attrName, 'desktop', 'parent-hover' )
+		}
+		if ( hasCollapsed ) {
+			valueDesktopCollapsed = getValue( attrName, 'desktop', 'collapsed' )
+		}
```

Comment:
This looks unnecessarily repeated. I would recommend just having `hasDesktop` to be true if `responsive === false`